src/components/home/Connected.jsx

```diff
@@ -1,7 +1,12 @@
 const Connected = () => {
   return (
     <div className="">
```

Comment:
you can apply flex justify center to the parent component here, without needing to put them into paragraph tags.